.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
```

Comment:
+1, by default we don't want to compile ES modules into commonjs, so tooling as webpack can do tree shaking. For testing & node.js there's `commonjs` babel env adding that.